flow-typed/npm/fixturez_vx.x.x.js

```diff
@@ -0,0 +1,18 @@
+// flow-typed signature: c0a247c13aec79d77053b130b7a57ff6
+// flow-typed version: <<STUB>>/fixturez_v1.1.0/flow_v0.110.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'fixturez'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'fixturez' {
+  declare module.exports: any;
```

Comment:
fixturez also ships flow types

Reply:
there are flow type errors and there is no way to fix them
could be a temporary fix

Reply:
at the end replaced by the type definition